map.js

```diff
@@ -0,0 +1,68 @@
+import L from "leaflet";
+import {
+  loadWeatherForecast,
+  failedForecast,
+  hideForecastLoader,
+} from "./domManipulation";
+import fetchWeatherForecast from "./api";
+
+const map = L.map("map").setView([-28.921631, 25.224609], 4);
```

Comment:
Why also do this if you call `loadMap` from your orchestrator. I don't understand.

Reply:
Oh I do now. `loadMap` doesn't actually instantiate your map. Maybe the naming is unclear.